core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt

```diff
@@ -326,63 +328,71 @@ fun LPGenericElevatedCard(
  */
 @Composable
 fun LPGenericElevatedCardImage(
-    imageUrl: String,
+    imageUrl: String?,
     title: String,
     description: String,
     onClick: () -> Unit
 ) {
+    val dp2 = LeasePertTheme.sizes.middleSize
+    val dp8 = LeasePertTheme.sizes.smallerSize
+    val dp12 = LeasePertTheme.sizes.midSmallSize
+    val dp175 = LeasePertTheme.sizes.extraSize175
     Card(
-        shape = MaterialTheme.shapes.medium.copy(all = CornerSize(12.dp)),
+        border = BorderStroke(width = dp2, color = Drab),
         colors = CardDefaults.cardColors(
-            containerColor = Drab,
+            containerColor = Bianca,
         ),
-        modifier = Modifier.padding(all = 10.dp)
+        shape = MaterialTheme.shapes.medium.copy(all = CornerSize(dp12)),
+        modifier = Modifier
+            .clickable(onClick = onClick)
+            .padding(all = dp8)
     ) {
-        Card(
-            colors = CardDefaults.cardColors(
-                containerColor = Bianca,
-            ),
-            shape = MaterialTheme.shapes.medium.copy(all = CornerSize(11.dp)),
-            modifier = Modifier
-                .width(177.dp)
-                .height(245.dp)
-                .clickable(onClick = onClick)
-                .padding(all = 1.2.dp)
-        ) {
-            Column {
-                Image(
-                    painter = rememberAsyncImagePainter(imageUrl),
+        Column {
+            if (imageUrl != null) {
+                SubcomposeAsyncImage(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .height(dp175),
+                    model = imageUrl,
+                    contentScale = ContentScale.Crop,
                     contentDescription = null,
+                    loading = {
+                        CircularProgressIndicator()
+                    }
+                )
+            } else {
+                AsyncImage(
                     modifier = Modifier
-                        .fillMaxWidth()
-                        .height(169.dp),
-                    contentScale = ContentScale.Crop
+                        .fillMaxSize()
+                        .height(dp175),
+                    model = "https://cdn-icons-png.flaticon.com/512/3792/3792702.png",
```

Comment:
I think it's better to use a local image because you couldn't display the image if you don't have internet

Reply:
Ready!